packages/flame/svg/Icons/icon-tipping.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><g id="tipping"><path id="base-1" d="M5 9H4l-3 2 2 4h1l2-1 2 1h1l6-3 1-1-1-1h-5l1 2-1 1-5-1v-1l5 1-1-1-4-2z"/><path id="base-2" d="M11 4a2 2 0 10-1 5 2 2 0 001-5zM6 1a2 2 0 100 4 2 2 0 000-4z" opacity=".7" fill-rule="evenodd" clip-rule="evenodd"/></g></svg>
```

Comment:
- We should not use any `opacity=".7"` here and find the closest gray/blue color. 
- Is it possible that it's also missing the `fill` attribute on the first path?

Reply:
I'm just saying these are Flame requirements. This has nothing to do with the icon export. **You** have to modify those properties inside the `SVG` code to respects those requirements. Like removing any path with opacity attribute to make sure a solid color is used.

Reply:
Not respecting these requirements might have an impact on the icon API not returning back the expected results when changing the color palette for example.

Reply:
Ok, apparently most of the icons use the base `opacity=".7"` on the level `detail-2`. Let's keep it this way then. It's just the background on which the icon is shown, might impact the actual color of the coins in this case.